### ISSUE TITLE
Optionally expose RandomBM recipes before the game is accepted

### ISSUE THREAD

**Issue by irilyth** [buttonmen-dev/buttonmen#2016]
Sometimes players might prefer to decline or withdraw a RandomBM game if they don't like the recipe (too weird, too boring, don't like Shadow dice, whatever). It'd be nice to support this by having the option to generate and expose the RandomBM recipe as soon as the game is created. Perhaps a checkbox on the create_game page?

**Comment by AdmiralJota** [comment]
And from a UI point of view, I think it would make sense to have the same checkbox control whether or not we reveal what button is selected when you have the site randomly choose an existing button.

**Comment by irilyth** [comment]
Yeah, that seems reasonable -- maybe the text would change depending on whether you selected Random Button or RandomBM, but either way, having one box in the same place for any "reveal buttons before accepting the game" feature seems good to me.

(For that matter, if anyone wanted this, it could also be used to hide the button choice if I was challenging you to a non-random game. I don't know how useful that would be, but if anyone does want that, probably it should use the same UI. :^)

**Comment by AdmiralJota** [comment]
That actually used to be an option on the old site! You could select it independently for each player. I remember sometimes seeing open games that were like "[Hidden Button From Brawl] vs. [Select a Button From Brawl]".

(Spoiler: It was usually Bennett.)
